apps/aecore/lib/aecore/account/account.ex

```diff
@@ -320,10 +321,11 @@ defmodule Aecore.Account.Account do
           non_neg_integer(),
           non_neg_integer(),
           non_neg_integer()
-        ) :: {:ok, SignedTx.t()} | {:error, reason()}
+        ) :: :ok | :error
   def build_tx(payload, tx_type, sender, sender_prv, fee, nonce, ttl \\ 0) do
     tx = DataTx.init(tx_type, payload, sender, fee, nonce, ttl)
-    SignedTx.sign_tx(tx, sender, sender_prv)
+    {:ok, tx} = SignedTx.sign_tx(tx, sender, sender_prv)
+    Pool.add_transaction(tx)
```

Comment:
You can still return the tx

Reply:
Ok, I'll return tx

Reply:
@DanielaIvanova the comments apply for all occurrences of this in the PR